configs/prod/roles/vulnerability.json

```diff
@@ -20,10 +21,11 @@
     },
     {
       "name": "Vulnerability viewer",
-      "description": "Read any Vulnerability resource.",
+      "description": "Perform read operations on any Vulnerability resource.",
       "system": true,
-      "platform_default": false,
-      "version": 3,
+      "platform_default": true,
+      "admin_default": false,
```

Comment:
this PR should contain only name + description changes, please if you want to change `platform_default` and `admin_default` flags, please create a separate PR, thank you